<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1193</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lbl_webcamfeed">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>561</width>
     <height>480</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_webcam_feed">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Web camp Feed</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_processed_webcamfeed">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>80</y>
     <width>561</width>
     <height>480</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>20</y>
     <width>1141</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_start">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Start Webcamp</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_stop">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Stop/Pause</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_canny">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Canny</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_detect">
      <property name="text">
       <string>Detect</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
